src/manage_nsfs/manage_nsfs_help_utils.js

```diff
@@ -185,23 +185,19 @@ Flags:
 
 const GLACIER_OPTIONS = `
 Usage:
-    manage_nsfs glacier <migrate | restore | expiry> [options]
-`;
+glacier <action> [options]
```

Comment:
minor comment - to match the style of the help.
```suggestion
glacier <action> [flags]
```